pkg/local_object_storage/blobstor/peapod/peapod.go

```diff
@@ -178,6 +178,11 @@ func (x *storage) Delete(prm common.DeletePrm) (common.DeleteRes, error) {
 func (x *storage) Iterate(prm common.IteratePrm) (common.IterateRes, error) {
 	var e error
 	err := x.ppd.Iterate(func(addr oid.Address, data []byte) bool {
+		data, e = x.compress.Decompress(data)
```

Comment:
Doesn't seem to be related to migration.

Reply:
fact, `Iterate` is just a method. What exactly do you mean by that?

Reply:
I mean it's placed in a commit that deals with migration, while it's not related to migration, this is the way `peapod` should operate.

Reply:
fixed